Dockerfile

```diff
@@ -1,15 +1,19 @@
-FROM node:16.20.0
+# Bullseyes uses Debian 11 with longer end-of-life date. Slim variant means smaller software component
+FROM node:16.20.0-bullseye-slim as build
 
-WORKDIR /usr/src/app
+RUN apt-get update && apt-get install -y --no-install-recommends dumb-init
+
+ENV NODE_ENV production
 
-COPY package*.json ./
```

Comment:
Can we omit that because it was always already copied with the second copy command?

Reply:
what do you mean? we only have one copy command in line 10 `COPY --chown=node:node . /usr/src/app`

Reply:
Yes, I was just wondering why we can delete this copy command. And I think that before we only had this to run npm install. However, the second `COPY . .` would have overwritten the package*.json files, correct?